src/components/20-molecules/top-content-bar/index.js

```diff
@@ -103,23 +121,88 @@ class AXATopContentBar extends InlineStyles {
   }
 
   render() {
-    const { variant } = this;
+    const {
+      variant,
+      icon,
+      stickymobile,
+      overlaydesktop,
+      closable,
+      closed,
+    } = this;
+
+    if (!closed) {
```

Comment:
Now - from reading lines 133-137 I can understand initiallyclosed (which does not participate in any logic elsewhere) - and prove that it is _not_ needed.

Here is a truth table of `closed` and the consequences for `render`:

closed   render
-----------------
false       content
true        empty

Thus, initiallyclosed can be eliminated in favour of <axa-top-content-bar closed ...>. 

Is there any situation where this alternative expression of the initially-closed state cannot be used?

If not, please delete initiallyclosed in code and documentation.

Reply:
initiallyclosed removed